dashboard/app/views/pd/workshop_dashboard/index.html.haml

```diff
@@ -0,0 +1,17 @@
+- content_for(:head) do
+  = stylesheet_link_tag 'css/pd', media: 'all'
+
+%script{type: "text/javascript", src: "https://maps.googleapis.com/maps/api/js?client=#{CDO.google_maps_client_id}&libraries=places"}
```

Comment:
I wonder if there's an npm module we could use instead (not something that we need to address immediately)